src/context/AppProvider.tsx

```diff
@@ -83,6 +96,14 @@ const initialState: State = {
   supportedChain: true,
   accountItem: null,
   online: true,
+  message: {
+    title: 'SAVE V2 has launched!',
+    subtitle:
+      'You’ll need to migrate your balance to continue earning interest.',
+    emoji: '🎉',
+    url: '#',
```

Comment:
Maybe this should link to save?

Reply:
For now this message will only show on /save - I was thinking of pointing this to a medium article closer to the time. I'll remove the button for now though whilst there's no url 👍